README.md

```diff
@@ -98,7 +98,7 @@ See the mashup's [readme](./mashups/smart-home/README.md).
 
 ### Using docker-compose
 
-You can start the devices inside a container, for that running `docker-compose up` at the root directory builds and runs the containers. For custom configuration, take a look at the `Dockerfile` of each device or [docker-compose.yml](./docker-compose.yml).
+You can start the devices inside a container, for that running `docker-compose up` at the root directory builds and runs the containers. For custom configuration, take a look at the `Dockerfile` of each device or `docker-compose.yml`.
```

Comment:
Hmm, as you have mentioned in the other PR, there is actually no `docker-compose.yml` file to be found anywhere in the repository (if I am not mistaken), so maybe this should actually refer to `docker-compose-things.yml` instead?

```suggestion
You can start the devices inside a container, for that running `docker-compose up` at the root directory builds and runs the containers. For custom configuration, take a look at the `Dockerfile` of each device or the `docker-compose-things.yml` file in the root directory.
```

Reply:
@egekorkan @hasanheroglu ?